crates/rattler_menuinst/src/lib.rs

```diff
@@ -173,6 +173,29 @@ pub fn install_menuitems(
     Ok(trackers)
 }
 
+/// Remove all menu items from a given prefix record.
+/// This function will remove the menu items from the system,
+/// and update the prefix record with removing the tracker entries
+pub fn remove_menuitems_for_record(
+    target_prefix: &Path,
+    prefix_record: &PrefixRecord,
```

Comment:
Maybe we should take it by value as it will be out of date when the function returns?

Or we could return the updated PrefixRecord.

Reply:
Used by value